examples/resources/gworkspace-connector.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   google_admin_email: <GOOGLE_WORKSPACE_ADMIN_EMAIL>
   google_service_account_uri: file:///var/lib/teleport/gworkspace-creds.json
   issuer_url: https://accounts.google.com
-  redirect_url: https://<cluster-url>:3080/v1/webapi/oidc/callback
+  redirect_url: https://mytenant.teleport.sh/v1/webapi/oidc/callback
```

Comment:
It's a little confusing to use teleport.example.com in some places and teleport.sh in other places.

Does this only appear in cloud contexts?

Reply:
Good point. @ptgott is this codified in a style guide somewhere?

Reply:
Right, but the page you linked here has separate tabs for cloud and self hosted, which is why I asked if this file is only referenced in cloud contexts.